tests/internal/e2e/test/test_utils/vcast_utils.ts

```diff
@@ -1230,3 +1253,62 @@ export async function selectItem(contentAssist: ContentAssist, item: string) {
 function normalizeContentAssistString(content: string): string {
   return content.replace(/⏎/g, "\n").replace(/\s+/g, " ").trim();
 }
+
+/**
+ * Function to read the last 'lineNumber' lines of the log file and check if the stringArray elements are present
+ * @param lineNumber Amount of line numbers to look at starting from the end of the log file
+ * @param stringArray List of strings that need to be contained withing last |lineNNumber|
+ * @returns true, if all strings are found, else false
+ */
+export async function requestInLogs(
```

Comment:
Maybe this should be `checkIfRequestInLog`?

Reply:
Changed in 3509429a2b77635f18359b6f01833aa8409d9dd9